modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1522,20 +1522,20 @@ private CompletableFuture<Void> dropTableAsyncInternal(String name) {
                 dropTblFut.completeExceptionally(new TableNotFoundException(DEFAULT_SCHEMA_NAME, name));
             } else {
                 tablesCfg.change(chg ->
-                        chg.changeTables(tblChg -> {
-                            TableView tableCfg = tblChg.get(name);
+                                chg.changeTables(tblChg -> {
+                                    TableView tableCfg = tblChg.get(name);
 
-                            if (tableCfg == null) {
-                                throw new TableNotFoundException(DEFAULT_SCHEMA_NAME, name);
-                            }
+                                    if (tableCfg == null) {
+                                        throw new TableNotFoundException(DEFAULT_SCHEMA_NAME, name);
+                                    }
 
-                            tblChg.delete(name);
-                        }).changeIndexes(idxChg -> {
-                            List<String> indicesNames = tablesCfg.indexes().value().namedListKeys();
+                                    tblChg.delete(name);
+                                }).changeIndexes(idxChg -> {
+                                    List<String> indicesNames = tablesCfg.indexes().value().namedListKeys();
```

Comment:
Since you already changed this code, can I ask you to use `chg` instead of `tablesCfg` and maybe extract `chg.indexes()` into a variable if necessary?